hive_agent/wallet/wallet.py

```diff
@@ -9,9 +10,15 @@
 from eth_account.signers.local import LocalAccount
 from eth_account.messages import encode_defunct
 
+def get_log_level():
```

Comment:
this function is duplicated around. move it to a `utils.py` file that can be imported by other modules